shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -0,0 +1,99 @@
+
+    try (MetaMasterClient client =
+        new RetryHandlingMetaMasterClient(MasterClientConfig.defaults())) {
+      Set<MasterInfoField> masterInfoFilter = new HashSet<>((Arrays
+          .asList(MasterInfoField.MASTER_ADDRESS, MasterInfoField.WEB_PORT,
+              MasterInfoField.RPC_PORT, MasterInfoField.START_TIME_MS,
+              MasterInfoField.UP_TIME_MS, MasterInfoField.VERSION,
+              MasterInfoField.SAFE_MODE)));
+      MasterInfo masterInfo = client.getMasterInfo(masterInfoFilter);
+
+      System.out.println("    Master Address: " + masterInfo.getMasterAddress());
+      System.out.println("    Web Port: " + masterInfo.getWebPort());
+      System.out.println("    Rpc Port: " + masterInfo.getRpcPort());
+      System.out.println("    Started: "
+          + CommonUtils.convertMsToDate(masterInfo.getStartTimeMs()));
+      System.out.println("    Uptime: "
+          + CommonUtils.convertMsToClockTime(masterInfo.getUpTimeMs()));
+      System.out.println("    Version: " + masterInfo.getVersion());
+      System.out.println("    Safe Mode: " + masterInfo.isSafeMode());
+    } catch (UnavailableException e) {
+      e.printStackTrace();
+      System.out.println("Please check the Alluxio master status.");
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+
+    try (RetryHandlingBlockMasterClient client =
+        new RetryHandlingBlockMasterClient(MasterClientConfig.defaults())) {
+      Set<BlockMasterInfoField> blockMasterInfoFilter = new HashSet<>(Arrays
+          .asList(BlockMasterInfoField.LIVE_WORKER_NUM, BlockMasterInfoField.LOST_WORKER_NUM,
+              BlockMasterInfoField.CAPACITY_BYTES, BlockMasterInfoField.USED_BYTES,
+              BlockMasterInfoField.FREE_BYTES, BlockMasterInfoField.CAPACITY_BYTES_ON_TIERS,
+              BlockMasterInfoField.USED_BYTES_ON_TIERS));
+      BlockMasterInfo blockMasterInfo = client.getBlockMasterInfo(blockMasterInfoFilter);
+
+      System.out.println("    Live workers: " + blockMasterInfo.getLiveWorkerNum());
+      System.out.println("    Lost workers: " + blockMasterInfo.getLostWorkerNum());
+
+      System.out.println("    Total Capacity: "
+          + FormatUtils.getSizeFromBytes(blockMasterInfo.getCapacityBytes()));
+      Map<String, Long> totalCapacityOnTiers = blockMasterInfo.getCapacityBytesOnTiers();
+      for (Map.Entry<String, Long> capacityBytesTier : totalCapacityOnTiers.entrySet()) {
+        System.out.println("        Tier: " + capacityBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(capacityBytesTier.getValue()));
+      }
+
+      System.out.println("    Used Capacity: "
+          + FormatUtils.getSizeFromBytes(blockMasterInfo.getUsedBytes()));
+      Map<String, Long> usedCapacityOnTiers = blockMasterInfo.getUsedBytesOnTiers();
+      for (Map.Entry<String, Long> usedBytesTier: usedCapacityOnTiers.entrySet()) {
+        System.out.println("        Tier: " + usedBytesTier.getKey()
```

Comment:
we need a better way to keep track of the indentation - it would be very easy for this to accidentally get out of line

Reply:
changed to use:

    int numOfSpaces = 4;
    String spaces = String.format("%" + numOfSpaces + "s", "");

    System.out.println(spaces + "Master Address: " + masterInfo.getMasterAddress());
    System.out.println(spaces + "Web Port: " + masterInfo.getWebPort());
    System.out.println(spaces + "Rpc Port: " + masterInfo.getRpcPort());

Any other better ways?

Reply:
if we maintain an `mIndentationLevel` field to keep track of the current indentation level, we could implement a `print` method like
```java
private void print(String text) {
  String indent = Strings.repeat(" ", mIndentationLevel * INDENT_SIZE);
  mPrintWriter.println(indent + text);
}
```

Reply:
Thanks,
that makes code look better and not that tricky.

```java
  private void print(String text) {
    String indent = Strings.repeat(" ", mIndentationLevel * INDENT_SIZE);
    System.out.println(indent + text);
  }
```